node/bitfield-signing/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "polkadot-node-bitfield-signing"
+version = "0.1.0"
+authors = ["Peter Goodspeed-Niklaus <peter.r.goodspeedniklaus@gmail.com>"]
```

Comment:
sorry for being late for, but we usually put
```toml
authors = ["Parity Technologies <admin@parity.io>"]
```

Reply:
Oh, good point; I'll fix that.

Reply:
#1459.